app/styles/newStash.scss

```diff
@@ -51,6 +51,11 @@ input {
   border: 1px solid #CCC;
 }
 
+#stash-description, input {
+  width: 85%;
+  // max-width: 210px;
```

Comment:
Are you considering going back to max-width 210?  If not consider removing the comment.  Other than that it looks good, I like the changes from let to const.